model/models.urdf.xacro

```diff
@@ -66,6 +66,51 @@
 		<origin xyz="0 0 0.0145" rpy="0 0 0"/>
 	</xacro:property>
 
+<!-- http://www.ati-ia.com/products/ft/ft_models.aspx?id=Gamma -->
+<xacro:property name="gamma">
+			<inertial>
+				<mass value="0.255"/>
+				<origin xyz="0 0 0.01665"  />
+				<xacro:cylinder_inertia_def radius="0.0377" length="0.0333" mass="0.255"/>
+			</inertial>
+			<visual>
+				<!-- origin xyz="0 0 0.00725" rpy="0 0 0"/ -->
+				<origin xyz="0 0 0" rpy="0 0 0"/>
+				<geometry>
+					<!-- cylinder radius="0.0085" length="0.0145"/ -->
+					<mesh filename="package://force_torque_sensor/model/meshes/nano17.stl" scale="4.4 4.4 1"/>
```

Comment:
Found it: http://www.3dcontentcentral.com/download-model.aspx?catalogid=201&id=280

But the coordinate system does not coincide with the URDF, I'll work it out and send the STL.